package.json

```diff
@@ -77,17 +76,15 @@
     "test": "jest",
     "clean": "ocular-clean all",
     "build": "ocular-clean all && ocular-build --dist esm && lerna run build",
-    "publish": "ocular-publish",
+    "publish": "npm run lerna:publish",
```

Comment:
nit: using `yarn` instead?

Reply:
I've tried this, but doesn't work -- probably overridden by the default behavior of `yarn publish` (publishing the root package instead of running a script called "publish"